src/Components/ComGallary.js

```diff
@@ -6,7 +6,12 @@ export const ComGallary = ({ imagesUrl }) => (
   <div className="clearfix">
     {
       imagesUrl.map((el, index) => (
-        <img key={index} src={`${path.resolve('images/asset')}${el}`} alt="" className="rounded float-left imgGallary" />
+        <img
+          key={index}
+          src={`${path.resolve('images/asset')}${el}`}
```

Comment:
why do you need `path` there? It's unnecessary dependency, just use `src={`images/asset/${el}`}`